src/Stryker.Core/Stryker.Core/Mutants/MutantStatus.cs

```diff
@@ -6,6 +6,7 @@ public enum MutantStatus
         Killed,
         Survived,
         Timeout,
+        NoCoverage,
```

Comment:
I would like this status to be added to the report as well

Reply:
Isn't it already? I've seen that the Json reporter just uses ToString on this enum. So when a mutant has status NoCoverage this will be the string that ends up in the json right?

Reply:
Woops, I was thinking of an older iteration of the report. I forgot I made the new one a lot prettier 😅